README.md

```diff
@@ -4,11 +4,32 @@ Library for validating Canadian Postal Codes and US Zip Codes
 
 # usage
 
-yarn add zip-postal-codes
+`yarn add zip-postal-codes`
 
+```javascript
+1. CA
+`import { validatePostal } from 'zip-postal-codes/src';`
+<strong> Province </strong>: should be the first 2 letters or the the full name of the province
+<strong> Postal Code </strong>: only the first letter 
+    validatePostal(province: string, postalCode: string); 
```

Comment:
I'm a bit confused on this. We are we mixing HTML and JavaScript code together? What does this render in Markdown?